.cspell.json

```diff
@@ -38,72 +57,111 @@
         "imsc",
         "insafe",
         "isnt",
+        "k",
         "Keio",
+        "kesteren",
+        "kongress",
         "ldquo",
         "linenums",
         "linkchecker",
         "linkhref",
         "Localise",
+        "mdash",
+        "media",
         "mediacapture",
         "memsub",
         "metaviewport",
         "middot",
         "munge",
+        "narayanan",
         "newurl",
         "nodate",
+        "noexistence",
         "nonlocal",
         "Normalise",
+        "npmjs",
         "Nresult",
         "nullptr",
         "OBSL",
         "ocument",
+        "Ohai",
         "ontents",
+        "parallelised",
         "patentpolicy",
+        "prevrecuri",
+        "prevversionuri",
         "Pubrules",
         "rdquo",
         "recognising",
         "rectrack",
+        "reprojection",
         "Resc",
+        "retargeted",
+        "retargeting",
         "RSCND",
         "sbyte",
         "secno",
         "secnos",
         "sels",
+        "sensical",
         "shortlink",
         "shortname",
         "shortnames",
         "sotd",
+        "sp",
         "sparql",
         "specberus",
         "SPSD",
         "stackalloc",
+        "stream",
         "strictfp",
         "subm",
+        "submlogo",
         "supersedable",
+        "tocxref",
         "ttml",
         "uage",
         "uarr",
         "ulong",
         "unneutral",
+        "unrecognize",
         "ushort",
         "vali",
-        "valide",
         "Valide",
+        "valide",
         "valuenow",
+        "vcard",
+        "vulns",
         "wantarray",
+        "WCAG",
         "wcag",
-        "wgnote",
+        "webauthn",
+        "WEBIDL",
+        "WEBRTC",
         "WG's",
-        "whut"
+        "wgnote",
+        "whacko",
+        "whut",
+        "widl",
+        "YYYYMMDD"
     ],
     "dictionaries": [
         "html",
         "js",
         "css",
+        "npm",
         "en-gb"
     ],
     "ignorePaths": [
+        "**/*.eot",
+        "**/*.ttf",
+        "**/*.woff",
+        "**/*.svg",
+        ".nyc_output/**",
+        "coverage/**",
+        "test/**",
```

Comment:
Took the easier way out by ignoring this folder instead of further building out the dictionary